ddtrace/tracer/tracer.go

```diff
@@ -605,11 +605,20 @@ func (t *tracer) sample(span *span) {
 	t.prioritySampling.apply(span)
 }
 
-func startExecutionTracerTask(ctx gocontext.Context, name string, spanID uint64) (gocontext.Context, func()) {
+func startExecutionTracerTask(ctx gocontext.Context, span *span) (gocontext.Context, func()) {
 	if !rt.IsEnabled() {
 		return ctx, func() {}
 	}
-	ctx, task := rt.NewTask(ctx, name)
-	rt.Log(ctx, "span id", strconv.FormatUint(spanID, 10))
+	// Task name is the resource (operationName) of the span, e.g.
+	// "POST /foo/bar" (http) or "/foo/pkg.Method" (grpc).
+	taskName := span.Resource
```

Comment:
```suggestion
	taskName := span.Name
```
nit: it looks like this could also just be the `span.Name` since it's the same as `span.Resource`, and might be a bit easier to understand when looking at the code.

Reply:
I'll leave it as-is for now because we also use `span.Resource` in `applyPPROFLabels`. But now you've got me curious - what's the supposed to be the difference between the two? Would they ever hold different values?

We'll definitely switch to use `span.Name` in both places if it makes more sense in the long run.